lib/config/default.js

```diff
@@ -934,5 +931,49 @@ exports.config = () => ({
      * loop data.
      */
     native_metrics: { enabled: true }
+  },
+
+  /**
+   *
+   * Controls the behavior of Logs in Context within agent
+   * TODO: sorting out some default values as the spec differs from conversations
```

Comment:
Noticing this TODO so will let that get hashed out.

Reply:
ah, i can remove this. thanks for calling it out